spdx/package.py

```diff
@@ -192,7 +192,12 @@ def calc_verif_code(self):
         hashes = []
 
         for file_entry in self.files:
-            hashes.append(file_entry.calc_chksum())
+            if isinstance(file_entry.chk_sum, checksum.Algorithm) and
```

Comment:
Note that this is not a valid syntax and the multline if should be wrapped in parens. I fixed this and pushed that to the LF git repo.